pubsub-client/src/nonblocking/pubsub_client.rs

```diff
@@ -543,7 +543,7 @@ impl PubsubClient {
 
                     // Get text from the message
                     let text = match msg {
-                        Message::Text(text) => text,
+                        Message::Text(text) => text.to_string(),
```

Comment:
nit: I don't think this is necessarily worth it, but we could avoid the string allocation here by pulling out `text` as it is (ie the old code) and using `text.as_str()` on line 558, and only clone the string if we need to. What do you think?